localisation/00_greymane.csv

```diff
@@ -0,0 +1,2 @@
+greymane;Blood of [This.GetFirstNameWithNick];Sang de [This.GetFirstNameWithNick];Blut von [This.GetFirstNameWithNick];;;;;;;;;;;x
+greymane_desc;Archibald Greymane was the king of Gilneas and the father of Genn Greymane. Under his rule, Gilneas prospered. His views on asking for help were that it was for the weak.;;;;;;;;;;;;;x
```

Comment:
What is a point of this file? These lines exist in `localisation/00_zuwc_united_localisation.csv`.

Reply:
i accidently uploaded the test file.

Reply:
Bang! The file still exists.